google-cloud-spanner/src/main/java/com/google/cloud/spanner/connection/StatementParser.java

```diff
@@ -360,7 +360,7 @@ public static String removeCommentsAndTrim(String sql) {
     final char HYPHEN = '-';
     final char DASH = '#';
     final char SLASH = '/';
-    final char ASTERIKS = '*';
+    final char ASTERISKS = '*';
```

Comment:
I think this should be `ASTERISK` (singular instead of plural)

Reply:
Thanks, fixed it.